### PR TITLE
Auth0

### PR DESCRIPTION
**Pull request by nick-brady**

### CODE REVIEW

File: src/screens/AuthScreen/index.js [hunk 1]
```diff
@@ -1,82 +1,87 @@
...
+  Text,
+  View
+} from 'react-native';
+import { Button, Screen } from '../../components';
+import Auth0 from 'react-native-auth0';
 import theme from '../../theme';
 import styles from './styles';
 
-export default class AuthScreen extends React.Component {
+var credentials = require('./auth0-credentials');
```
Comment (smitch88) [inline src/screens/AuthScreen/index.js]: import syntax

File: src/screens/AuthScreen/index.js [hunk 2]
```diff
@@ -1,82 +1,87 @@
...
-    if(this.props.navigation.state.params){
-      console.log('current username!', this.props.navigation.state.params.username);
-      this.setState({ username: this.props.navigation.state.params.username})
-    }
-  }
-
-  render(){
-    const { error, username, isAuthenticating } = this.state;
-    const loadingDisplay = isAuthenticating ? 'flex' : 'none';
+  onLogin = () => {
```
Comment (smitch88) [inline src/screens/AuthScreen/index.js]: Why did you remove the syntax that already handled the promise?

You also need to do this in the `databroker` code and add in another method like `authByAuth0` you are breaking the other parts of the app with this change. The `databroker` is what holds the token to make future calls.

File: src/screens/AuthScreen/auth0-credentials.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
```
Comment (smitch88) [inline src/screens/AuthScreen/auth0-credentials.js]: This isn't on the node side you can use es6 export syntax. Babel is already being used.

File: src/screens/AuthScreen/index.js (hunk 2 above)
Comment (nick-brady) [reply src/screens/AuthScreen/index.js]: fixed

**Pull request merged**